pkg/distributor/distributor.go

```diff
@@ -756,21 +767,28 @@ func (d *Distributor) prePushHaDedupeMiddleware(next push.Func) push.Func {
 		}
 
 		numSamples := 0
+		numHistograms := 0
```

Comment:
We can drop this variable, and just use `numSamples`.

Reply:
done